src/codegen/metadata/script_info.py

```diff
@@ -28,5 +28,123 @@
             "relativeOutputPath": "_task_modules\\channels\\do_channel.py", 
             "templateFile": "_task_modules\\channels\\do_channel.py.mako",
         },
+    ],
+    "modulesToCopy":[
+        {
+            "relativeOutputPath": "__init__.py",
```

Comment:
That's right both are same now, but I just have that since I wanted to get some suggestions on where the handwritten files can be placed in the src folder.  Based on this we can plan to remove the repeated field.

Reply:
The logic has been updated to have mako files for handwritten files